src/io/src/OutNtupleProc.cc

```diff
@@ -467,7 +467,10 @@ Processor::Result OutNtupleProc::DSEvent(DS::Root *ds) {
     outputTree->Fill();
   }
   if (options.untriggered && ds->GetEVCount() == 0) {
+    // EV information
     evid = -1;
+    subev = -1;
+    nhits = -1;
```

Comment:
Thanks for catching this! Do you think nhits should be set to 0 instead of -1?

Reply:
i think -1 is safest as an invalid value for this int object.  
though seemingly unlikely, one could imagine a case where there are actually 0 hits in a detector while a trigger is still issued for some reason.

Reply:
ok, fine. I guess downstream can also override that behavior if needed. LGTM!